### PR TITLE
Fix #412: next button on first screen after installation

### LINKED ISSUE #412: next button on first screen after installation
For the current scenario, when a user downloads the app, or when data is cleared, the first activity asks the user to enter the name. but there is no next button here. the only way to proceed from here is to click on the tick from the keyboard. so I propose to give a next button which takes the user to the tutorial sliders and hence no change in the flow. 
 I'd like to add this thing.

### PR DESCRIPTION
**Pull request by DravitLochan**

### CODE REVIEW

File: source-code/app/src/main/java/org/buildmlearn/toolkit/activity/FirstRunActivity.java [hunk 1]
```diff
@@ -44,11 +45,37 @@ protected void onCreate(Bundle savedInstanceState) {
         }
         setContentView(R.layout.activity_first_run);
 
-
+        next = (Button) findViewById(R.id.button_next);
         findViewById(R.id.focus_thief).clearFocus();
-        Animation animBounceinup=AnimationUtils.loadAnimation(getBaseContext(),R.anim.bounceinup);
+        Animation anim_bounceinup=AnimationUtils.loadAnimation(getBaseContext(),R.anim.bounceinup);
```
Comment (yadav-rahul) [inline source-code/app/src/main/java/org/buildmlearn/toolkit/activity/FirstRunActivity.java]: @DravitLochan Why using `Snake case` here? `Java` variables should be named with `lowerCamelCase` .

**Comment by DravitLochan** [comment]
I don't remember using snake case but I'll fix it right away. :)

File: source-code/app/src/main/java/org/buildmlearn/toolkit/activity/FirstRunActivity.java (hunk 1 above)
Comment (DravitLochan) [reply source-code/app/src/main/java/org/buildmlearn/toolkit/activity/FirstRunActivity.java]: Ok. On reviewing the project, it came to my notice that at many places snake cases have been used. At nowhere I used it. What I propose is as sson as I get green flag from Travis and cadacy bots,  I'll create another issue clearing it all over. If that is okay?

**Comment by DravitLochan** [comment]
@opticod  can you please see and tell me why travis is continuously failing me? :(

**Comment by m-murad** [comment]
@DravitLochan 
The error is
` No resource found that matches the given name (at 'text' with value '@string/button_next').`
You haven't added the button text string to `string.xml`

**Comment by DravitLochan** [comment]
ohho. damn! just give me a sec

**Comment by DravitLochan** [comment]
@opticod  please review and merge. Thank you for your support!

**Comment by m-murad** [comment]
@DravitLochan 
You might wanna squash the commits.

**Comment by DravitLochan** [comment]
done! @opticod  @free4murad

**Comment by DravitLochan** [comment]
@free4murad  @opticod :( 

why does it want next to be a local variable instead?

**Comment by opticod** [comment]
@DravitLochan , we follow snake case for ids and camelCase for variable names. We will be updating the README section for easiness of contributors. 

> why does it want next to be a local variable instead?

If we doesn't require a variable globally, then we can limit the scope only where it is required. It helps to optimise function activation stacks calls by the program.

**Comment by DravitLochan** [comment]
@opticod no issues. I made the pr proper. please review and act :)

**Comment by DravitLochan** [comment]
@opticod did you review the pr after the changes?

**Review by opticod — COMMENTED** [review]
@DravitLochan, I reviewed just now. 
There were some changes required that I added following your commit. 

1. Refactoring of your code was missing
2. ```anim_bounceinup``` violates naming convention. 
3. ```next``` button should start with capital character.

**Comment by opticod** [comment]
@DravitLochan, you can go through 2c79128 , and see the changes that I made in addition to you.

**Comment by DravitLochan** [comment]
Oh thanks for the changes 😊

**Pull request merged**